### PR TITLE
Added Introductory Problems Directory

### PR DESCRIPTION
**Pull request by Zeo-shark**
Add a description of changes here.

### CODE REVIEW

File: Introductory Problems/ZeoShark_Weird_Algorithm1.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//Author: Sourav Bera
+
+//Result: https://cses.fi/problemset/result/1065843/
+import java.util.*;
+public class ZeoShark_Weird_Algorithm1{
+
+    public static void main(String args[]){
+        Scanner in = new Scanner(System.in);
+        long n= in.nextInt();
+        while(n!=1)
```
Comment (anushreeBera) [inline Introductory Problems/ZeoShark_Weird_Algorithm1.java]: Please give appropriate spacing between operators and operand: while(n != 1)

File: Introductory Problems/Zeoshark_BitStrings.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+public class Zeoshark_BitStrings {
+    public static void main(String args[] ) {
+        Scanner in= new Scanner(System.in);
```
Comment (anushreeBera) [inline Introductory Problems/Zeoshark_BitStrings.java]: Spacing on either hands of assignment operator is uneven.

**Comment by anushreeBera** [comment]
Please update coding convention

**Comment by Zeo-shark** [comment]
> Please update coding convention

Updated code abiding coding conventions

**Review by anushreeBera — CHANGES_REQUESTED** [review]

File: Introductory Problems/ZeoShark_Weird_Algorithm1.java (hunk 1 above)
Comment (Zeo-shark) [reply Introductory Problems/ZeoShark_Weird_Algorithm1.java]: add appropriate spacing between operator and operands

File: Introductory Problems/Zeoshark_BitStrings.java (hunk 2 above)
Comment (Zeo-shark) [reply Introductory Problems/Zeoshark_BitStrings.java]: add spacing in either side of the operator

File: Introductory Problems/WeirdAlgorithm1.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        Scanner in = new Scanner(System.in);
+        long n = in.nextInt();
+        while(n != 1)
+        {
+            System.out.print(n+" ");
+            if(n%2 == 0)
+                n = n/2;
+            else
+                n = (n*3)+ 1;
+        }
```
Comment (Zeo-shark) [inline Introductory Problems/WeirdAlgorithm1.java:25]: **Learn to add inline comments + drag**

**Pull request merged**